### PR TITLE
formatting dockerfiles for 2.0.0

### PR DESCRIPTION
**Pull request by Elizaaaaa**
*Issue #, if available:*

*Description of changes:*
formated dockerfiles
unpinned awscli version
pinned numpy to a higher version

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docker/2.0.0/py3/Dockerfile.cpu [hunk 1]
```diff
@@ -17,101 +32,91 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
...
+    pandas==0.24.2 \
+    Pillow==6.1.0 \
+    h5py==2.9.0 \
+    keras_applications==1.0.8 \
+    keras_preprocessing==1.1.0 \
+    keras==2.3.1 \
+    requests==2.22.0 \
+    awscli==1.16.196 \
+    mpi4py==3.0.2 \
+    # "sagemaker-tensorflow>=1.14,<1.15" && \
```
Comment (sboshin) [inline docker/2.0.0/py3/Dockerfile.cpu]: I believe they released the sagemaker-tensorflow==2.0.0.1.0.0

Comment (nadiaya) [reply docker/2.0.0/py3/Dockerfile.cpu]: yup

File: docker/2.0.0/py2/Dockerfile.cpu [hunk 2]
```diff
@@ -69,41 +71,48 @@ RUN apt-get update && apt-get install -y \
...
+ENV PYTHONIOENCODING=UTF-8 
+ENV LANG=C.UTF-8 
+ENV LC_ALL=C.UTF-8
 
-ARG framework_support_installable=sagemaker_tensorflow_container-2.0.8.dev0.tar.gz
-ARG sagemaker_tensorflow_extensions=tensorflow-2.0.0-cp27-cp27mu-manylinux2010_x86_64.whl
-COPY $framework_support_installable .
-COPY $sagemaker_tensorflow_extensions .
+ARG FRAMEWORK_SUPPORT_INSTALLABLE=sagemaker_tensorflow_container-2.0.8.dev0.tar.gz
+ARG THENSORFLOW_WHL=tensorflow-2.0.0-cp27-cp27mu-manylinux2010_x86_64.whl
```
Comment (saimidu) [inline docker/2.0.0/py2/Dockerfile.cpu]: nit: typo

File: docker/2.0.0/py3/Dockerfile.gpu [hunk 3]
```diff
@@ -95,35 +98,43 @@ ENV PATH /usr/local/openmpi/bin/:$PATH
...
+RUN ln -s $(which ${PYTHON}) /usr/local/bin/python \
+ && ln -s $(which ${PIP}) /usr/bin/pip
 
-ARG framework_support_installable=sagemaker_tensorflow_container-2.0.8.dev0.tar.gz
-COPY $framework_support_installable .
-COPY $sagemaker_tensorflow_extensions .
+# Use THENSORFLOW_WHL instead of TF_URL before releasing
+# ARG TF_URL="https://tensorflow-aws.s3-us-west-2.amazonaws.com/1.14/AmazonLinux/gpu/final/tensorflow-1.14.0-cp36-cp36m-linux_x86_64.whl"
+
+ARG FRAMEWORK_SUPPORT_INSTALLABLE=sagemaker_tensorflow_container-2.0.8.dev0.tar.gz
```
Comment (saimidu) [inline docker/2.0.0/py3/Dockerfile.gpu]: Define ARGs and ENVs at the top if they don't need to be defined/updated only after installation of a package.

File: docker/2.0.0/py3/Dockerfile.cpu (hunk 1 above)
Comment (Elizaaaaa) [reply docker/2.0.0/py3/Dockerfile.cpu]: added

File: docker/2.0.0/py2/Dockerfile.cpu (hunk 2 above)
Comment (Elizaaaaa) [reply docker/2.0.0/py2/Dockerfile.cpu]: done

File: docker/2.0.0/py3/Dockerfile.gpu (hunk 3 above)
Comment (Elizaaaaa) [reply docker/2.0.0/py3/Dockerfile.gpu]: moved

**Pull request merged**